src/styles.css

```diff
@@ -138,7 +154,7 @@
 
 .card {
   flex-direction: column-reverse;
-  min-height: 75vh;
+  min-height: 60vh;
 }
 
 .card_info {
```

Comment:
Please, remove empty CSS rules in the project to not confuse yourself,

Reply:
thanks for reviewing, can you specify please wich rule is empty?